cli/error.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"gopkg.in/go-playground/validator.v9"
+)
+
+var (
+	ErrCreateNewUser          = errors.New("failed to create a new user")
+	ErrCreateNewNamespace     = errors.New("failed to create a new namespace")
+	ErrDuplicateUser          = errors.New("user already exists")
+	ErrDuplicateNamespace     = errors.New("namespace already exists")
+	ErrChangePassword         = errors.New("failed to reset the user password")
+	ErrUserNotFound           = errors.New("user not found")
+	ErrNamespaceNotFound      = errors.New("namespace not found")
+	ErrFailedAddNamespaceUser = errors.New("failed to add the namespace for the user")
+	ErrFailedDeleteUser       = errors.New("failed to delete the user")
+	ErrFailedDeleteNamespace  = errors.New("failed to delete the namespace")
+	ErrFailedUpdateUser       = errors.New("failed to reset the password for the user")
+	ErrFailedRemoveMember     = errors.New("failed to remove member from the namespace")
+	ErrFound                  = errors.New("")
```

Comment:
Where this one is used?

Reply:
Just a generic error. This is used on the service.